### ISSUE TITLE
Get error in console if user cancels the attack roll

### ISSUE THREAD

**Issue by kaelad02** [ElfFriend-DnD/foundryvtt-attack-roll-check-5e#2]
When you click on the Attack button in the item chat card it pops up a dialog to select options for the roll (i.e. a situational bonus and adv/dis). If you click on the Close button on the upper-right of that dialog it will cancel the attack roll. When I do that I get an error in the console:

```
TypeError: Cannot read property 'options' of null
[Detected 4 packages: attack-roll-check-5e, lib-wrapper, more-hooks-5e, system:dnd5e]
    at _checkAttackRoll (module.js:37)
    at Function._call (eval at <anonymous> (listeners.js:91), <anonymous>:4:14)
    at Function.callAll (foundry.js:153)
    at Item5e.rollAttackPatch (rollAttack.js:13)
    at async Function._onChatCardAction (entity.js:1395)
```

In case it matters, I'm not using roll modules like Better Rolls or Midi QOL.

**Comment by akrigline** [comment]
Thanks for this report! I'm thinking this might be a problem in a few of my hooks from more hooks 5e.

I've seen something similar with auto check save, will investigate.

**Comment by akrigline** [comment]
I believe this is fixed with More Hooks 5e's 0.4.1 release
